docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -483,7 +483,8 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``tz_convert``              | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``tz_localize``             | N                               |                                  |                                                    |
+| ``tz_localize``             | P                               | ``axis``, ``level``, ``copy``,   |                                                    |
+|                             |                                 | ``ambiguous``, ``nonexistent``   |                                                    |
```

Comment:
qq: how did we determine what parameters are going to be skipped (not implemented)?  what's the process / criteria?

Reply:
Good question. Initially, we want to make sure we are supporting all the default values (for parameters that do have default values). The parameter `tz` here for instance does not have a default value, so it has to be fully supported.

We can later add more parameter coverage, which usually happens based on customer requests. The rationale is that  default values are typically the most commonly used by users.